media/.gitkeep

```diff
@@ -0,0 +1 @@
+
```

Comment:
is this file necessary anymore since you have the logo now?

Reply:
ah looks like I created it initially for uploading :D .. will remove